deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml

```diff
@@ -38,6 +38,13 @@ env: |
     value: "individual-index-v1"
   - name: ES_TRANSFORMER_PI_INDEX
     value: "transform-ifms-pi-index"
+  - name: KAFKA_BROKER
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: kafka-brokers
+  - name: TRANSFORMER_PI_KAFKA_TOPIC
```

Comment:
@prerna-egov please add conditional values, we can not update that according to enviornment.

Reply:
@shailesh-egov made the changes. please check

Reply:
@prerna-egov your if is not in the correct place. Please check the given example and change it accordingly
https://github.com/odisha-muktasoft/DIGIT-DevOps/blob/mukta-1.22-changes/deploy-as-code/helm/charts/digit-works/utilities/ifms-cronjob-pa/values.yaml#L39

Reply:
@shailesh-egov I took refernce from here https://github.com/egovernments/DIGIT-DevOps/blob/unified-env/deploy-as-code/helm/charts/common/values.yaml#L41